packages/picker/stories/picker.stories.ts

```diff
@@ -34,8 +34,10 @@ export default {
         invalid: false,
         open: false,
         quiet: false,
+        pending: false,
     },
     argTypes: {
+        ...argTypes,
```

Comment:
Would you do us the favor while you're touching Picker and add an `options` array to https://github.com/adobe/spectrum-web-components/blob/razvancir96/add-picker-loading-state/packages/picker/stories/args.ts#L29 as a sibling to the `controls` property that listed `options: [ 'Small', 'Medium', 'Large', 'Extra Large' ]`?